UnityProject/Assets/Scripts/Player/PlayerSprites.cs

```diff
@@ -182,68 +174,55 @@ public void SubSetBodyPart(BodyPart Body_Part, string path)
 
 		if (customisationStorage != null)
 		{
-			bool pass = true;
+			var data = customisationStorage.Data.Replace("@£", "\"");
```

Comment:
a good point here is that you need customisation for organs to

Reply:
there arent many changes on how sprites are handled, mostly its just moving stuff around, like in this case. If the concern is customization for organs (beards, hair, eyes, etc) its working as previously